test/test_c10d.py

```diff
@@ -2967,6 +2967,8 @@ def _test_broadcast_coalesced(self, process_group, device):
     def _run_all_reduce(self, pg):
         pg.allreduce(torch.rand(10).cuda(self.rank))
 
+    # Need 2.4+ for error checking.
+    @requires_nccl_version(2400)
```

Comment:
This check should be flipped with `requires_nccl`.

Could you add the comment to the skip check as argument, so it shows up in the skip description?